src/js/actions/guides.js

```diff
@@ -66,13 +75,20 @@ define(function (require, exports) {
                 range: "guide",
                 index: 1,
                 count: -1
-            },
-            getOpts = {
-                failOnMissingProperty: true
             };
 
-        // FIXME: Should we reverse these?
-        return descriptor.getPropertiesRange(docRef, rangeOpts, _guideProperties, getOpts);
+        var requiredPromise = descriptor.getPropertiesRange(docRef, rangeOpts,
```

Comment:
I wonder if it would make sense to factor this pattern out into a helper function? I assume this is basically copied and pasted from the layer range properties code?